shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
@fortMaximus well then, I guess there are two steps now:

1. Make sure you cherry-pick that commit to the feature branch (see second image) and then push it. This should be enough for you to keep working.
2. Delete that commit from master branch. What you did should have worked, but there's probably something I'm not grasping. Can you see that commit in `git log --oneline`? or does it appear in `git reflog`?

The first image above should picture your current status, doesn't it?